docs/janssen-server/reference/json/feature-flags/janssenauthserver-feature-flags.md

```
@@ -8,37 +8,52 @@ tags:
-| REVOKE_SESSION | Enable/Disable session revocation endpoint | [Details](#revoke_session) |
 | REVOKE_TOKEN | Enable/Disable token revocation endpoint | [Details](#revoke_token) |
 | SSA | Enable/Disable Software Statement Assertion(SSA) feature | [Details](#ssa) |
 | STAT | Enable/Disable Stat service | [Details](#stat) |
+| STATUS_LIST | Enable/Disable status list endpoint | [Details](#status_list) |
 | STATUS_SESSION | Enable/Disable session status check endpoint | [Details](#status_session) |
 | U2F | Enable/Disable support for Universal 2nd Factor(U2F) protocol | [Details](#u2f) |
 | UMA | Enable/Disable support for User-Managed Access (UMA) | [Details](#uma) |
 | USERINFO | Enable/Disable OpenID Connect [userinfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo) | [Details](#userinfo) |
 
 
+### ACCESS_EVALUATION
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)